### PR TITLE
#157731626 Add support for macrocycle, mesocycle and microcycle

### PR DESCRIPTION
**Pull request by tibetegya**
### What does this PR do?
Adds support for macrocycle, mesocycle and microcycle
#### Description of Task to be completed?
1. Run Autopep8 on the code base
#### How should this be manually tested?
1. Run the commands below
```
$ git clone https://github.com/andela/wg-rogue-one.git

$ git checkout ft-support-macrocycle-mesocycle-157731626

```
follow the setup instructions in [README.rst](https://github.com/andela/wg-rogue-one/blob/ft-support-macrocycle-mesocycle-157731626/README.rst)
2. Run the app with `python manage.py runserver`
3. Login to wger 
4. Navigate to the `Workout overview` and add a new workout. 
5. You should be able to choose a training cycle. 
screen shot 2018-06-22 at 12 43 49

6. Choose between `Microcycle`, `Mesocycle`and `Macrocycle`.
7. screen shot 2018-06-22 at 11 59 53
Note: Added workouts should have a cycle indicator

#### Any background context you want to provide?
Macro is an annual plan, meso is phase of training with a duration of between 2-6 weeks and micro is instead only a week.

This is needed for sports periodization ( powerlifting, weightlifting, strongman, ecc ) or at least to create a fitness program drafted on more than only one week.

#### What are the relevant pivotal tracker stories?
[#157731626](https://www.pivotaltracker.com/story/show/157731626)

### CODE REVIEW

File: wger/manager/templates/workout/view.html [hunk 1]
```diff
@@ -58,25 +58,59 @@
...
+{% endif %}
+
 {% for day in workout.canonical_representation.day_list %}
     <div id="div-day-{{ day.obj.id }}">
         {% render_day day.obj is_owner %}
     </div>
 {% empty %}
     {% if is_owner %}
-        <p>
+        <!-- <p>
```
Comment (peterwade153) [inline wger/manager/templates/workout/view.html]: commented out code?

Comment (tibetegya) [reply wger/manager/templates/workout/view.html]: thanks

**Review by dondrzzy — COMMENTED** [review]
Last line of the file

File: wger/manager/views/day.py [hunk 2]
```diff
@@ -96,7 +106,7 @@ class DayCreateView(DayView, CreateView):
     Generic view to add a new exercise day
     '''
 
-    title = ugettext_lazy('Add workout day')
+    # title = ugettext_lazy('Add workout day')
```
Comment (hariclerry) [inline wger/manager/views/day.py]: I would rather you remove this line of commented code, it will make the code look neat.

**Review by hariclerry — CHANGES_REQUESTED** [review]

File: wger/manager/views/day.py (hunk 2 above)
Comment (tibetegya) [reply wger/manager/views/day.py]: thank you for that i have removed it

**Comment by peterwade153** [comment]
Looks good now

**Review by hariclerry — APPROVED** [review]

**Comment by emugaya** [comment]
@tibetegya Fix test Coverage.

**Pull request merged**